src/api/spec/models/workflow/step/configure_repositories_spec.rb

```diff
@@ -62,7 +66,6 @@
 
       before do
         target_project
-        project
```

Comment:
`project` is renamed to `path_project1`, but still, initializing it here isn't needed. It's already initialized when `path_repository1` is initialized.